pkg/kubelet/kubelet.go

```diff
@@ -3030,6 +3033,12 @@ func (kl *Kubelet) setNodeStatusImages(node *api.Node) {
 	if err != nil {
 		glog.Errorf("Error getting image list: %v", err)
 	} else {
+		// sort the images from max to min, and only set top N images into the node status.
+		sort.Sort(ByImageSize(containerImages))
+		if maxImagesInNodeStatus < len(containerImages) {
+			containerImages = containerImages[0 : maxImagesInNodeStatus-1]
```

Comment:
Actually, this returns `maxImagesInNodeStatus-1` images, is that what you want? @resouer